Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -924,6 +935,10 @@ static void webkitWebViewConstructed(GObject* object)
             priv->display = wpe_display_get_default();
         }
     }
+#else
```

Comment:
This would leave out the WPEPlatform case when both `USE_LIBWPE` and `ENABLE_WPE_PLATFORM` are enabled. I think this should be:


```suggestion
#endif

#if ENABLE(WPE_PLATFORM)
```

Reply:
No, this is already inside a ENABLE(WPE_PLATFORM) block.